sagemaker-datawrangler/joined-dataflow/explore_data.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
<p>Write some descriptions for what you are doing in the cell (i.e. import the necessary packages, etc.).</p><p> </p><p>Also, maybe it is best to move this to the "Prerequisites: Get Data" section since it is used there?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>